backend/requirements.txt

```diff
@@ -0,0 +1,5 @@
+Django==2.1.2
```

Comment:
We already have requirements directory with base and dev requirements. This file is redundant.

Reply:
My bad...